tutorials/scripting/gdextension/gdextension_cpp_example.rst

```diff
@@ -363,7 +363,7 @@ loaded for each platform and the entry function for the module. It is called ``g
     [configuration]
 
     entry_symbol = "example_library_init"
-    compatibility_minimum = "4.1"
+    compatibility_minimum = "4.2"
```

Comment:
This change is not necessary to do as it's the minimum compatibility (which was introduced in 4.1) for the extension to work and doesn't need an update.

Reply:
Reverted. Thanks for pointing it out!